app/models/survey.rb

```diff
@@ -19,6 +19,8 @@ class Survey < ActiveRecord::Base
 
   belongs_to :first_question, foreign_key: :first_question_id, class_name: 'Question'
 
+  self.per_page = 10
```

Comment:
the pagination gem will look for this variable when determining how many surveys each page should contain.